components/relevancy/src/schema.rs

```diff
@@ -22,6 +22,14 @@ pub const SQL: &str = "
         interest_code INTEGER NOT NULL,
         PRIMARY KEY (url_hash, interest_code)
     ) WITHOUT ROWID;
+
+    -- Stores user interest vectors.  The `kind` field stores the raw code from the `InterestVectorKind` enum.
+    CREATE TABLE user_interest(
+        kind INTEGER NOT NULL,
```

Comment:
Unrelated to this PR, just for discussion.

For debugging, what do you think about adding some dimension tables for `Kind` and `Interests`? We currently store those in their numeric forms in `url_interest` and `user_interest` which is fine as we can programmatically convert them into the more readable string representations. But when we query the db directly, we will have to manually do the conversion. With dimension tables, we can easily get the string forms via table joins.

Reply:
Sounds good, we can leave it for now.

A few cases I found it useful:

When I updated [this](https://mozilla-hub.atlassian.net/wiki/spaces/CLOUDSERVICES/pages/695173233/Suggest+Interaction+data+and+Implementation) to reflect the in-flight suggestion / keyword counts, I usually just pull data directly from `suggest.sqlite`. Since we only store provider IDs in the `suggestion` table, I have to open up the source code to manually map provider IDs to the actual strings. Likewise, when I query `relevancy.sqlite` to get the `interest -> url-counts` mapping, only interest codes are available and I have to look up the interest enum in the source to decode them.